doc/source/building/index.rst

```diff
@@ -386,30 +386,14 @@ interface is self-documenting, so please see ``spin --help`` and
 
 .. admonition:: IDE support & editable installs
 
-    While the ``spin`` interface is our recommended way of working on SciPy,
+    While the ``spin build`` interface is our recommended way of working on SciPy,
     it has one limitation: because of the custom install location, SciPy
-    installed using ``spin`` will not be recognized automatically within an
+    will not be recognized automatically within an
     IDE (e.g., for running a script via a "run" button, or setting breakpoints
     visually). This will work better with an *in-place build* (or "editable
     install").
 
-    Editable installs are supported. It is important to understand that **you
-    may use either an editable install or spin in a given repository clone,
-    but not both**. If you use editable installs, you have to use ``pytest``
-    and other development tools directly instead of using ``spin``.
-
-    To use an editable install, ensure you start from a clean repository (run
-    ``git clean -xdf`` if you've built with ``spin`` before) and have all
-    dependencies set up correctly as described higher up on this page. Then
-    do::
-
-        # Note: the --no-build-isolation is important! meson-python will
-        # auto-rebuild each time SciPy is imported by the Python interpreter.
-        pip install -e . --no-build-isolation
-
-        # To run the tests for, e.g., the `scipy.linalg` module:
-        pytest scipy/linalg
-
+    Editable installs are supported via ``spin install``.
     When making changes to SciPy code, including to compiled code, there is no
     need to manually rebuild or reinstall. When you run ``git clean -xdf``,
```

Comment:
```suggestion
    need to manually rebuild or reinstall. However, should you need to run ``git clean -xdf``,
```
It may be confusing to the reader to see this right after the statement that no manual rebuild/reinstall is needed. Let's make it clear that the user doesn't necessarily ever need to use `git clean -xfd` - but *if* they do, they should `pip uninstall scipy` and `spin install` again.